README.md

```diff
@@ -137,14 +137,38 @@ juju relate temporal-worker-k8s admin/cos.loki
 juju relate temporal-worker-k8s admin/cos.prometheus
 ```
 
-
 ```bash
 # Access grafana with username "admin" and password:
 juju run grafana/0 -m cos get-admin-password --wait 1m
 # Grafana is listening on port 3000 of the app ip address.
 # Dashboard can be accessed under "Temporal Worker SDK Metrics", make sure to select the juju model which contains your Temporal worker operator charm.
 ```
 
+## Vault
+
+The Temporal worker operator charm can be related to the
+[Vault operator charm](https://charmhub.io/vault-k8s) to securely store
+credentials that can be accessed by workflows. This is the recommended way of
+storing workflow-related credentials in production environments. To enable this,
+run the following commands:
+
+```bash
+juju deploy vault-k8s --channel edge
```

Comment:
Is this the right channel? I see on [charmhub](https://charmhub.io/vault-k8s) the following,
![Screenshot from 2024-06-13 08-43-14](https://github.com/canonical/temporal-worker-k8s-operator/assets/46668016/969e87c4-4dab-43f6-8866-8baead5e45a0)

Reply:
Good catch, updated to `1.15/edge`.